Jenkinsfile

```diff
@@ -0,0 +1,12 @@
+wrappedNode(label: 'linux && x86_64', cleanWorkspace: true) {
```

Comment:
Specifying a label of "docker-stable" would get you a newer AMI with 17.06.0 (and very soon 17.06.1) if you'd like.